docs/document/docs/getting-started/quick-start.md

```diff
@@ -191,6 +166,31 @@ Output Result：
 
 ![button customize](/img/button_customize.png)
 
+### Use the CDN
+:::tip
+See the guidance and sample code at [Use with downloading kuc.min.js file](#use-with-downloading-kucminjs-file) section.
+:::
+
+1. Add the following CDN URL into the `JavaScript and CSS Customization` of a deployed Kintone app. ([Customizing an App with JavaScript and CSS](https://get.kintone.help/k/en/user/app_settings/js_customize.html))<br/>
+Once the CDN is being imported to the app, you will have access to the global object of `Kuc`.
+
+   - If you want to import a particular version, specify the version number after the project name.
+     ```bash
+     https://unpkg.com/kintone-ui-component@1.0.0/umd/kuc.min.js
+     ```
+
+   - For the latest version of Kintone UI Component:
+     ```bash
+     https://unpkg.com/kintone-ui-component/umd/kuc.min.js
+     ```
+
+2. The implementation details and steps provided below are aligned with those described in the "Use with downloading kuc.min.js file" section.
+
+:::caution
+unpkg is not supported by Cybozu. We recommend you to use it for verification.<br/>
+In a production environment, it is advisable to use a specific version by downloading the kuc.min.js file to avoid the possible effects of unpkg technical obstacle and glitches.
```

Comment:
unpkg is not a CDN service provided by Cybozu. It is recommended that you use this for verification.<br/>
In the production environment, you can download the `kuc.min.js` and use it to avoid any failures and problems related to unpkg.